package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "angular2-jwt",
+  "name": "@auth0/angular-jwt",
```

Comment:
Just to let you know, this package.json file doesn't concern the library itself, but the test host application, so we don't have to give it the same name as the lib, this might even cause problems while npm-linking the lib while testing locally, the lib's package.json file is in `projects/angular-jwt/package.json`